airbyte-integrations/connectors/source-tidb/src/main/java/io/airbyte/integrations/source/tidb/TiDBSourceOperations.java

```diff
@@ -0,0 +1,69 @@
+package io.airbyte.integrations.source.tidb;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.mysql.cj.MysqlType;
+import com.mysql.cj.jdbc.result.ResultSetMetaData;
+import com.mysql.cj.result.Field;
+import io.airbyte.db.SourceOperations;
+import io.airbyte.integrations.source.mysql.MySqlSourceOperations;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+import static io.airbyte.db.jdbc.JdbcConstants.*;
```

Comment:
Please convert to single import here.

Reply:
Thanks for you review. I've changed it.
[convert to single import](https://github.com/airbytehq/airbyte/pull/11283/commits/167fed07cd98db54e16beb7d778a2ead78d3818f)